bucket/pyzo.json

```diff
@@ -0,0 +1,44 @@
+{
+    "version": "4.11.2",
+    "description": "The Interactive editor for scientific Python",
+    "homepage": "https://pyzo.org/",
+    "license": "BSD-3-Clause",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/pyzo/pyzo/releases/download/v4.11.2/pyzo-4.11.2-win64.zip",
+            "hash": "435cfb5ee617c30dac5fc7880a41c0046c3375067e7b24da3e7e0f121099ec6a",
+            "extract_dir": "pyzo-4.11.2",
+            "extract_to": "pyzo-4.11.2-win64"
+        },
+        "32bit": {
+            "url": "https://github.com/pyzo/pyzo/releases/download/v4.11.2/pyzo-4.11.2-win32.zip",
+            "hash": "9be1a82b428bc74c1c395f401d473054612eb383d7dd3c352d54324b57e39390",
+            "extract_dir": "pyzo-4.11.2",
+            "extract_to": "pyzo-4.11.2-win32"
+        }
+    },
+    "bin": "pyzo.exe",
+    "shortcuts": [
+        [
+            "pyzo.exe",
+            "Pyzo"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/pyzo/pyzo"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/pyzo/pyzo/releases/download/v$version/pyzo-$version-win64.zip",
+                "extract_dir": "pyzo-$version",
+                "extract_to": "pyzo-$version-win64"
+            },
+            "32bit": {
+                "url": "https://github.com/pyzo/pyzo/releases/download/v$version/pyzo-$version-win32.zip",
+                "extract_dir": "pyzo-$version",
+                "extract_to": "pyzo-$version-win32"
```

Comment:
Why is extract-to required?

Reply:
I don't that it is necessary. I added extract-to following the samples I studied. One thought is that since extract-dir on it's own is same for 32 and 64bit ensuring extract-to uses different names would allow side-by-side installs. I don't see a situation where I would do this myself though. I'm happy to remove it if this if it isn't standard practice.

Reply:
It is not